test/jdk/sun/misc/UnsafeMemoryAccessWarnings.java

```diff
@@ -43,19 +43,15 @@
 class UnsafeMemoryAccessWarnings {
 
     /**
-     * Test default is "allow"
+     * Test default is "warn"
      */
-    @Test
-    void testDefault() throws Exception {
-        test("allocateMemory+freeMemory+objectFieldOffset+putLong+getLong+invokeCleaner")
-            .shouldHaveExitValue(0)
-            .shouldNotContain("WARNING: A terminally deprecated method in sun.misc.Unsafe has been called")
-            .shouldNotContain("WARNING: sun.misc.Unsafe::allocateMemory")
-            .shouldNotContain("WARNING: sun.misc.Unsafe::freeMemory")
-            .shouldNotContain("WARNING: sun.misc.Unsafe::objectFieldOffset")
-            .shouldNotContain("WARNING: sun.misc.Unsafe::putLong")
-            .shouldNotContain("WARNING: sun.misc.Unsafe::getLong")
-            .shouldNotContain("WARNING: sun.misc.Unsafe::invokeCleaner");
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "allocateMemory+freeMemory",
+            "objectFieldOffset+putLong+getLong"
```

Comment:
Is the removal of `invokeCleaner` method name from these method names intentional?

Reply:
Yes because testOneWarning is only useful when called with two or more method names.

Reply:
Sorry I wasn't clear - what I meant was including `invokeCleaner` in the long list of method names like it was previously - `allocateMemory+freeMemory+objectFieldOffset+putLong+getLong+invokeCleaner`, not as an individual method name on its own. It's not a big deal to leave it out though, since these are not the complete set of method names anyway.